purchase_mass_mail/models/purchase_order.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2023 Moduon Team S.L.
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl-3.0)
+
+
+from odoo import models
+
+
+class PurchaseOrder(models.Model):
+    _inherit = "purchase.order"
+
+    def action_all_rfq_related_send(self):
+        template = self.env.ref("purchase.email_template_edi_purchase", False)
+        try:
+            compose_form = self.env.ref(
+                "mail.email_compose_message_wizard_form",
+                False,
+            )
+        except ValueError:
+            compose_form = False
+        ctx = dict(
+            self.env.context,
+            default_model="purchase.order",
+            active_model="purchase.order",
+            active_id=self.ids[0],
```

Comment:
```suggestion
            active_id=self.id,
```

Reply:
Hi @yajo but this solution don't work correctly in the case wich you have selected some records, return singleton error :)